client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -95,6 +114,26 @@ const ProfilePhoto: React.FC<ProfilePhotoProps> = ({ header, currentUser, curren
           >
             {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Upload a file from your device'}
           </Button>
+          <br></br>
+          <Button
+            sx={{
+              marginTop: '40px',
+            }}
+            onClick={() => {
```

Comment:
It is not recommended to use inline function defs as there is speculation that it affects performance in React